src/layers/src/geojson-layer/geojson-utils.ts

```diff
@@ -120,24 +116,19 @@ export function getGeojsonDataMaps(dataContainer: any, getFeature: GetFeature):
  */
 export function parseGeometryFromString(geoString: string): Feature | null {
   let parsedGeo;
-
-  // try parse as geojson string
-  // {"type":"Polygon","coordinates":[[[-74.158491,40.83594]]]}
-  try {
-    parsedGeo = JSON.parse(geoString);
-  } catch (e) {
-    // keep trying to parse
-  }
-
-  // try parse as wkt
-  if (!parsedGeo) {
+  if (geoString.length && geoString[0] === '{') {
```

Comment:
I am not exactly sure when this function is used, but generally this would not be a safe change., Strings may start with spaces and sometimes JSON geometry can be an array. 

Might be better to check if the first char is an uppercase letter and if so attempt to parse as WKT.

Reply:
Great, will rewrite it 🙌